### ISSUE TITLE
DS1: Scenario-Based System Test 1

### ISSUE THREAD

**Issue by mlin1026** [CSE-110-Winter-2024/project-team-17#17]
As a developer I want to test the whole application so that I can make sure the app is working.

Scenario 1:
First Andrew tap open the app on the device
Then, his mom calls him over to help with the dishes that he was supposed to wash yesterday. He quickly locks his phone and heads downstairs to do the dishes before he gets yet another scolding. Once he is done, he returns back to his room and unlocks his phone, where the app is waiting for him ([US 7](https://docs.google.com/document/d/1zKQFf3yiS8dFTIRljFRsDln9VseDl9d3rzvaqYm7Gqg/edit#heading=h.d1ri2yw4x7sz)).
The app displays “there is not existing task for the day”. ([US 2](https://docs.google.com/document/d/1zKQFf3yiS8dFTIRljFRsDln9VseDl9d3rzvaqYm7Gqg/edit#heading=h.ch8m7b54upg8)) Phew, time to relax! 
Then, he receives a message from his friend, Natalie. “Are you done with your CSE110 reading quiz? It’s due 12.30pm today!”. Not a relaxing day it seems. Andrew then returns to the app and looks at the time. It’s only 9am, he still has time! ([US1](https://docs.google.com/document/d/1zKQFf3yiS8dFTIRljFRsDln9VseDl9d3rzvaqYm7Gqg/edit#heading=h.b2ugluex687k))
Andrew then presses the “+” sign and intends to add a new task. ([US3](https://docs.google.com/document/d/1zKQFf3yiS8dFTIRljFRsDln9VseDl9d3rzvaqYm7Gqg/edit#heading=h.tvz2y64qh88e), [US4](https://docs.google.com/document/d/1zKQFf3yiS8dFTIRljFRsDln9VseDl9d3rzvaqYm7Gqg/edit#heading=h.k9a84f898fyd))
However, his keyboard is not working! “Nevermind”, he thought. Let me add it using my voice. He presses on the microphone and says “CSE110 HW”. The task is added. ([US5](https://docs.google.com/document/d/1zKQFf3yiS8dFTIRljFRsDln9VseDl9d3rzvaqYm7Gqg/edit#heading=h.k9a84f898fyd))
Since Andrew’s new 2024 resolution is to not procrastinate, he decides to just do his assignment anyway. After he is done, he returns to the app to press on the existing task to “finish it”. A strikethrough on the task is shown. Hooray! ([US8](https://docs.google.com/document/d/1zKQFf3yiS8dFTIRljFRsDln9VseDl9d3rzvaqYm7Gqg/edit#heading=h.b7c8v9l3vbq1))
He remembers that it is Natalie’s birthday tomorrow. In order to not forget to send his good friend a birthday message, he added yet another task (“Wish Natalie a Happy Birthday”) to the list, knowing that the task will stay until tomorrow. 
The next day, when Andrew opens the app, true enough, the task (“Wish Natalie a Happy Birthday”) is still present ([US6](https://docs.google.com/document/d/1zKQFf3yiS8dFTIRljFRsDln9VseDl9d3rzvaqYm7Gqg/edit#heading=h.oww03txgef6c)). 
While using the app, Andrew accidentally pressed on the task and a strikethrough the task is shown even though he has not completed the task. Fat fingers! ([US8](https://docs.google.com/document/d/1zKQFf3yiS8dFTIRljFRsDln9VseDl9d3rzvaqYm7Gqg/edit#heading=h.b7c8v9l3vbq1))
He tried to undo his actions by pressing on the finished task and thankfully, the strikethough was gone. The task is back. ([US9](https://docs.google.com/document/d/1zKQFf3yiS8dFTIRljFRsDln9VseDl9d3rzvaqYm7Gqg/edit#heading=h.581rlgmk7gfb))
After thinking long and hard, he sent a very sincere birthday message to Natalie and can finally finish the task. He pressed on the task and the strikethrough is back, this time for real. ([US8](https://docs.google.com/document/d/1zKQFf3yiS8dFTIRljFRsDln9VseDl9d3rzvaqYm7Gqg/edit#heading=h.b7c8v9l3vbq1)) What a day!
